lib/iris/io/__init__.py

```diff
@@ -227,6 +227,22 @@ def load_http(urls, callback):
             yield cube
 
 
+def _dot_save(cube, target):
```

Comment:
`iris.fileformats.dot.save` doesn't take any keyword arguments, so perhaps better to just get rid of them from the call a couple of lines further down from here.